src/parser/rogue/assassination/modules/talents/ElaboratePlanning.js

```diff
@@ -20,11 +21,16 @@ class ElaboratePlanning extends Analyzer {
   constructor(...args) {
     super(...args);
     this.active = this.selectedCombatant.hasTalent(SPELLS.ELABORATE_PLANNING_TALENT.id);
+    if (!this.active) {
+      return;
+    }
+    ABILITIES_AFFECTED_BY_DAMAGE_INCREASES.forEach(e => {
+      this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(e), this.addBonusDamageIfBuffed);
```

Comment:
Since `ABILITIES_AFFECTED_BY_DAMAGE_INCREASES` is array of spell objects itself, you should be able to do just 
```javascript
this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(ABILITIES_AFFECTED_BY_DAMAGE_INCREASES), this.addBonusDamageIfBuffed);
```
without the loop.